.github/workflows/lint-and-test.yml

```diff
@@ -15,7 +15,9 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Helm
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.9.0 # Also update in publish.yaml
```

Comment:
The latest should equal v3.10.1. Do we want to stick with fallback?

Reply:
@pdrastil I'm basing v3.9.0 on [this](https://github.com/Azure/setup-helm/blob/b70d33f56dce95f7970896b826403d7c1bbe8a22/src/run.ts#L17) but not opposed to 3.10.1

Reply:
updated to 3.10.1